app/models/dataset.rb

```diff
@@ -61,43 +61,22 @@ def publish!
   # What we actually want to index in Elastic, rather than the whole
   # dataset.
   def as_indexed_json(_options = {})
-    as_json(
-      only: %i[
-                name
```

Comment:
By having one attribute per line we keep the diff to just that line when we add or remove an attribute.

Reply:
But it makes the file so long :-( I notice we don't take this approach anywhere with attr_xxx, so it's somewhat consistent with that.

Reply:
That's why you use Vim. I agree that the convention isn't universally adopted, but I think it's a good one to adopt as we go along.

Reply:
Hmm...not sure how a particular editor helps with this. In my previous job, we usually managed to keep each file below a page in length. For me, it's helpful to be able to clearly see that a file in one go as much as possible.

Reply:
As this is mainly stylistic, perhaps we should get a third opinion - I'm happy to go with a majority.

Reply:
I can't read that crammed onto four lines :(

Reply:
I guess the alternative should be to move this code into it's own presenter, which is it's proper place. But not now.